.circleci/config.yml

```diff
@@ -248,6 +248,7 @@ jobs:
       GANACHE_CACHE_FOLDER: ../../.ganache-deployments
       GANACHE_PORT: 8547
       NODE_ENV: test
+      LOG_FILE: /tmp/xstate-wallet.log
```

Comment:
How about `XSTATE_WALLET_LOG_FILE`? If we don't plan to reuse this var for all packages and pipe their logs into the same file, having a more specific name makes easier to understand.

Reply:
I think we should be piping logs into the same file. Having logs from multiple sources in a single file, with "close-enough" proximity of log statements, seems more useful than having multiple files that require cross-referencing.

As long as there is a time stamp and an origin marker (`name` in xstate-wallet as of this PR) it doesn't matter that much, since you can either filter-by-name to split into multiple files, or you can concat and sort-by-timestamp to combine multiple files.